### ISSUE TITLE
Doesn't work for Greek language

### ISSUE THREAD

**Issue by anpapadakis** [agilord/diacritic#7]
Sample Text:

"Όλοι οι άνθρωποι γεννιούνται ελεύθεροι και ίσοι στην αξιοπρέπεια"

**Comment by isoos** [comment]
Yeah, not sure what we should do here, or in similar cases like Cyrillic alphabets. Any suggestion?
